examples/build_all.sh

```diff
@@ -2,6 +2,8 @@
 
 NUM_JOBS=$(nproc 2>/dev/null || sysctl -n hw.ncpu 2>/dev/null || 4)
 
+export TOCK_BUILDALL=1
```

Comment:
What is the meaning of this argument in a way that is meaningful to someone reading this script, without the context of the specific narrow use in the sensor-lorawan example Makefile, such that they could evaluate, e.g.: whether to have it on; whether it's being used correctly; whether it's ok to remove it because they're coming back to this in a year and it's unclear whether/why it was ever used; etc...

Reply:
@ppannuto You added this can you help explain here

Reply:
Added an explanation and example.